src/wcs1_1/src/test/java/org/geoserver/wcs/DescribeCoverageTest.java

```diff
@@ -171,6 +188,37 @@ public void testDescribeDemCoverageKvp() throws Exception {
         checkDemCoverageDescription(dom);
     }
 
+    @Test
+    public void testDescribeNoEnvelopeSRS() throws Exception {
+        Document dom =
+                getAsDOM(
+                        BASEPATH
+                                + "?request=DescribeCoverage&service=WCS&version=1.1.1&identifiers="
+                                + getLayerId(NO_ENVELOPE_SRS));
+        print(dom);
```

Comment:
We normally comment out these guys. They do not dump anything during the maven build, but show up in a run from IDE (which can be annoying as we have test classes with many tests, due to the high start-up time of each).